### DISCUSSION TITLE
Login Issues

### DISCUSSION THREAD

**Discussion by ghost** [telekom-security/tpotce#786] [Q&A]
Hello 

I have successfully installed Tpot 20. However, during login, despite typing the correct password it says incorrect and I'm unable to get in. Kindly assist

![image](https://user-images.githubusercontent.com/73940000/110279301-e6785a80-7ffe-11eb-9a54-69f1229c85cd.png)

**Comment by 2shagai** [comment]
I've same issue. Did you resolve this?
